dashboard/app/controllers/openai_chat_controller.rb

```diff
@@ -8,7 +8,17 @@ def chat_completion
       return render(status: :bad_request, json: {})
     end
 
-    messages = params[:messages]
+    if has_level_id_param?
+      level_id = params[:levelId]
+      level = Level.find(level_id)
+      test_file = level.validation["PainterPlusTest.java"]["text"]
+      messages = params[:messages]
+      messages.first["content"] = messages.first["content"] + " The contents of the test file are: #{test_file}"
+      messages.second["content"] = "The student's code is: " + messages.second["content"]
```

Comment:
This is a way-down-the-road problem, but I'm curious about how/if we can handle translations for system prompts, student code, etc...